### ISSUE TITLE
Multiple Issues and Suggestions

### ISSUE THREAD

**Issue by yamenzk** [frappe/print_designer#80]
Hello!
First of all, amazing work on this. I was really surprised how capable this designer is in generating print formats that appear almost similarly in the designer, print and pdf. You can see below my work on a purchase order print format.

![Screenshot_20231109_200605_Adobe Acrobat~2](https://github.com/frappe/print_designer/assets/98694759/9e1bad43-1667-4b79-b36d-20aa1e7cb3a8)
In the screenshot shown above, you can see my first attempt in using the print designer and it looks great! However, I have had few issues and suggestions during the process and would like to share them below.

- [ ] 1. In the items table, I currently have 1 item which renders properly as designed. However, lets say in an order of 20+ items, the table does not push the content further below to the next page, it just goes over or under other elements (depending on the Z-index). 
- [ ] 2. The image_view for items does not work (no image is shown)
- [ ] 3. "order by", "order to" and "purchase invoice" are all static elements and changing the color works on the designed but not in pdf/print
- [ ] 4. Page number and total pages only display in black text color
- [ ] 5. There is no tax breakdown table to display each item and the tax rate applied to it.
- [ ] 6. In "Company Address" these fields are not fetched correctly, I have created my company and set my billing, shipping and office addresses. However it does not get any information from them.
- [ ] 7. You can not set a table border to 0. It works in the designer but not in PDF/Print
- [ ] 8. Copy, Cut, Paste, Undo, Redo
- [ ] 9. Sometimes the color selection window gets stuck and you have to save and refresh the builder to get it away.
- [ ] 10. HTML and Jinja blocks and tags
- [ ] 11. Include alpha in color selection. 
- [ ] 12. Grid lines
- [ ] 13. Multiple pages print format.
- [ ] 14. Conditional display of elements and table columns
- [ ] 15. Display ✅ and ❎ instead of 1 and 0

**Comment by maharshivpatel** [comment]
@yamenzk 
### can you share your format like this :
You can paste the data here ( change any personal info )

https://github.com/frappe/print_designer/assets/39730881/b7b761c7-a2b1-43d4-a3b3-0dde25ba9699

### Comments on your points ( Issue 1 == Answer 1 )

1. click on header/footer button and set it accordingly then it will not overlap ( [Link](https://www.youtube.com/watch?v=XpmrLzbianE&t=747s) ) .
2.  haven't added yet, will add
3. There is a bug in Color control I will fix this ( If you share your format i can easily debug this ) Try to select Inner Text like [this](https://youtu.be/XpmrLzbianE?feature=shared&t=297) and  change it a color .
4. Same Issue as 3
5. Didn't Understand Where is it not available
6.  Please Share Example for this
7.  Yeah, that is wkhtmltopdf related issue you need to disable border from 
8.

**Comment by yamenzk** [comment]
I will reply to the rest as soon as I get back and I will share the format

**Comment by maharshivpatel** [comment]
> Currently away from my PC but I can't understand the solution to #1.
> 
> So let's consider creating a purchase order document.
> 
> The first section is company and supplier
> 
> Second section is item order table (item, qty, rate, amount)
> 
> Third section includes grand, net total, in words, etc..
> 
> 
> 
> For such example, the video you provided proposes a solution.
> 
> However let's say I want to add another item table, now it includes specifications like image, description, brand, material type and other custom fields concerning and item specification.
> 
> Then I want to add a payment terms table and/or a tax breakdown table 
> 
> 
> 
> Where do I place these tables? Part of the footer?

Yes, other elements that come after main table goes into footer ( this will repeat them on all pages ) if you don't want that you have to wait until i find a solution to introduce dynamic height containers

**Comment by maharshivpatel** [comment]
> I will reply to the rest as soon as I get back and I will share the format

@yamenzk I look forward to it

**Comment by SvbZ3r0** [comment]
@maharshivpatel 

- [ ] Dark mode behaviour is a wonky. Random blocks of text lose visibility when reloading (possibly because of white text on white background)

- [ ] Unicode is broken
![image](https://github.com/frappe/print_designer/assets/57280279/6d1b2144-7a25-4dd9-bdec-4880ede501bd)
while trying to display `Pad | Beige | ⌀40 cm`

- [ ] Accidentally clicking on a `Link` item has caused me to lose progress on an unsaved format many many times. Either links should lead to a new page or there should be some sort of session memory so progress is not lost.

**Comment by maharshivpatel** [comment]
No 1. is solved by #107
